revampedClient/components/cards/Card.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../assets/components/_card.scss';
+
+const Card = (props) => {
```

Comment:
I think jsDoc will make sense here for the sake of documentation

Reply:
will add that. 👍